packages/puppeteer-core/src/api/BrowserContext.ts

```diff
@@ -174,26 +174,6 @@ export abstract class BrowserContext extends EventEmitter<BrowserContextEvents>
    */
   abstract pages(): Promise<Page[]>;
 
-  /**
-   * Whether this {@link BrowserContext | browser context} is incognito.
-   *
-   * In Chrome, the
-   * {@link Browser.defaultBrowserContext | default browser context} is the only
-   * non-incognito browser context.
-   *
-   * @deprecated In Chrome, the
```

Comment:
let's move the description about what type of contexts and how many are created in Firefox and in Chrome to the documentation of the BrowserContext class